app/views/admin/creator_settings/new.html.erb

```diff
@@ -12,7 +12,7 @@
       <% end %>
     </div>
 
-    <%= form_tag(admin_creator_settings_path, method: "post", class: "relative z-elevate p-4") do %>
+    <%= form_tag(admin_creator_settings_path, method: "post", class: "relative z-elevate p-4", "data-action": "submit->creator-settings#formValidations") do %>
```

Comment:
We should not be able to submit the form if the brand color contrast is low, this function caters for this.